.github/workflows/deploy.yml

```diff
@@ -35,8 +35,9 @@ jobs:
 
     - name: Deploy to AWS CloudFormation
       run: |
+        sed "s/$REPLACE_ME$/$(openssl rand -hex 4)/" cloudformation.yaml > cloudformation-replaced.yaml
```

Comment:
`.yml`, not `.yaml`

Reply:
not in love with the dollar signs, makes me think they're some special character in the sed command, rather than literal characters to replace

Reply:
and we want to update `api/cloudformation.yml` rather than the top-level template, which complicates things

Reply:
since it's a throwaway environment, could do something like `sed -i "s/.../.../" **/cloudformation.yml`